std/range/package.d

```diff
@@ -9806,7 +9806,7 @@ if (isInputRange!Range)
     {
         return predFun(rhs, lhs);
     }
-    private Range _input;
+    /*private*/ Range _input;
```

Comment:
That's not really a proper fix and we shouldn't revert to such hacks :/

Reply:
Why does making it public make it safe?

Reply:
I just tried this locally and if I set `@trusted:` to `std.container.slist`, I can compile + test std.algorithm.iteration successfully with -dip1000 (e.g. `pmake std/algorithm/iteration.test`)
-> So I suggest to drop this specific change.

Reply:
@wilzbach Done as suggested